lib/color/color.cpp

```diff
@@ -0,0 +1,52 @@
+/** \file
+ * color.cpp
+ *
+ *  Created on: Apr 15, 2018
+ *      Author: itto
+ */
+
+#include "color.h"
+#include <opencv2/contrib/contrib.hpp>
+#include <time.h>
+
+using namespace cv;
+
+namespace MouseTrack {
+
+std::vector<double> GenerateRandomColor() {
+  // Random variable
+  // TODO add parameter for number of clusters that will determine how many
+  // colors to output
+  srand(time(NULL));
```

Comment:
To color clusters, I need a way to deterministically get a color for a certain cluster. Could we change the function signature to `GenerateRandomColor(int clusterIndex, int totalClusters)`? Maybe you can use `clusterIndex` and `totalClsuters` as seeds for srand? Otherwise seems ok.

Reply:
Thinking about it, removing the srand() statement should be fine, we don't need to pass in any arguments.